cmd/dlv/dlv_test.go

```diff
@@ -22,13 +22,22 @@ func assertNoError(err error, t testing.TB, s string) {
 
 func TestBuild(t *testing.T) {
 	const listenAddr = "localhost:40573"
-	cmd := exec.Command("go", "build", "github.com/derekparker/delve/cmd/dlv")
+	args := []string{"build"}
```

Comment:
Thinking about this, all of our CI platforms have `make`, we should just invoke `make build` here instead of trying to duplicate platform specific building rules here.